### PR TITLE
Fix partitioning raid for new UI changes

### PR DESCRIPTION
**Pull request by jknphy**
Fix partitioning raid for new UI changes. Overview now is selected by default and now there is a new flow for RAID being able to create several partitions inside one raid.

- Related ticket: https://progress.opensuse.org/issues/41849
- Needles: 
  - [~~needles openSUSE~~](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/445)
  - [~~needles SLE~~](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/957)
  - [~~needles openSUSE Part 2~~](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/446)
  - [~~needles SLE Part 2~~](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/958)
- Verification run:
  - [opensuse-15.1:S:C-Staging-RAID1](http://dhcp42.suse.cz/tests/203#step/partitioning_raid/267)
  - [Tumbleweed-RAID1](http://dhcp42.suse.cz/tests/202#step/partitioning_raid/250)
  - [Leap15.1-RAID1](http://dhcp42.suse.cz/tests/204#step/partitioning_raid/270)
  - [sle15sp1-RAID1](http://dhcp42.suse.cz/tests/205#step/partitioning_raid/271)

### CODE REVIEW

**Comment by okurz** [comment]
needles merged

File: tests/installation/partitioning_raid.pm [hunk 1]
```diff
@@ -31,6 +31,11 @@ sub addpart {
     else                         { die 'Unknown argument'; }
 
     assert_screen "expert-partitioner";
+    unless (is_sle('<15') || is_leap('<15.1') || is_tumbleweed) {
+        $cmd{addpart} = 'alt-r';
+        send_key 'alt-p';
+        wait_still_screen;
```
Comment (okurz) [inline tests/installation/partitioning_raid.pm]: do we really need to invest 7 seconds stilltime?

File: tests/installation/partitioning_raid.pm [hunk 2]
```diff
@@ -97,9 +102,16 @@ sub addraid {
         type_string "\t$chunksize";
     }
     send_key $cmd{next};
+    unless (is_sle('<15') || is_leap('<15.1') || is_tumbleweed) {
+        wait_still_screen;
```
Comment (okurz) [inline tests/installation/partitioning_raid.pm]: as above

**Comment by jknphy** [comment]
One additional needle to avoid one of the `wait_still_screen`:  [needles openSUSE](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/446) and [needles SLE](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/958)
VR updated.

File: tests/installation/partitioning_raid.pm [hunk 3]
```diff
@@ -31,6 +31,11 @@ sub addpart {
     else                         { die 'Unknown argument'; }
 
     assert_screen "expert-partitioner";
+    unless (is_sle('<15') || is_leap('<15.1') || is_tumbleweed) {
```
Comment (rwx788) [inline tests/installation/partitioning_raid.pm]: Could we at least put this monster condition in one place, so when it reaches TW, we don't have to change it in 3 places or potentially more?
Other than that LGTM.

**Pull request merged**